ruby/src/api.cc

```diff
@@ -235,6 +237,7 @@ namespace leatherman { namespace ruby {
     string api::to_string(VALUE v) const
     {
         v = rb_funcall(v, rb_intern("to_s"), 0);
+        v = rb_str_encode(v, utf8_value("UTF-8"), 0, _nil);
```

Comment:
I think we should assume UTF-8 internally and if there are any C++ libraries or system API interactions that require changing the encoding we can handle those locally.